### DISCUSSION TITLE
Feature vectors

### DISCUSSION THREAD

**Discussion by matroskin-the-cat** [open-mmlab/mmpose#2806] [Q&A]
How do I extract the feature vectors for a detection?
I want to use the feature vectors so that I can find similar poses.

**Comment by huimouyelu** [comment]
不懂为什么新闻上那么多鸡肋文章

**Reply by Ben-Louis** [reply]
您好，如果您想讨论与关键点检测相关的文章，可以另外开一个 discussion，并使用文明用语进行讨论

**Comment by Ben-Louis** [comment]
Hello @matroskin-the-cat, if you want to extract the feature vector of the pose estimator, you might need to make some changes to the `predict` method of the head class that you are using in your model. Additionally, it might be more convenient to use the keypoint coordinates to find similar poses.
